index.html

```diff
@@ -124,6 +124,89 @@ <h2><dfn>CropTarget Production</dfn></h2>
     </section>
     <section id="restricting-a-track">
       <h2><dfn>Restriction Mechanism</dfn></h2>
+      <section>
+        <h3>Definitions</h3>
+        <section>
+          <h4>Restrictable tracks</h4>
+          <p>
+            We say that a {{MediaStreamTrack}} <var>T</var> is a
+            <dfn>restrictable MediaStreamTrack</dfn> if and only if it fulfils all of the following
+            conditions:
+          </p>
+          <ul>
+            <li>
+              <var>T</var> is either a {{MediaStreamTrack}} returned inside of a {{MediaStream}}
+              through a call to {{MediaDevices/getDisplayMedia()}}, or a clone of such a track.
+            </li>
+            <li>
+              <var>T</var>.<a data-cite="mediacapture-streams/#dfn-kind">kind</a> is
+              <a data-cite="mediacapture-streams/#dfn-video">"video"</a>.
+            </li>
+            <li>
+              <var>T</var>.<a data-cite="mediacapture-streams/#dfn-readystate">readyState</a> is
+              <a data-cite="mediacapture-streams/#idl-def-MediaStreamTrackState.live">"live"</a>.
+            </li>
+            <li><var>T</var> is not <a data-cite="mediacapture-region#dfn-cropped">cropped</a>.</li>
+          </ul>
+        </section>
+        <section>
+          <h4>Elements eligible for restriction</h4>
+          <p>
+            We say that an {{Element}} <var>E</var> is <dfn>eligible for restriction</dfn> if and
+            only if it fulfils all of the following conditions:
+          </p>
+          <ul>
+            <li><var>E</var> forms a stacking context.</li>
+            <li>
+              <var>E</var> is
+              <a data-cite="css-transforms-2#grouping-property-values">flattened in 3D.</a>
+            </li>
+            <li>
+              <var>E</var> forms a <a data-cite="filter-effects-2/#backdrop-root">backdrop root</a>.
+            </li>
```

Comment:
It's worth adding a snippet of CSS that authors are expected to use to ensure these are met. Something like:

```css
#target {
  isolation: isolate; /* for stacking context */
  transform-style: flat; /* flattening */
}
```

I don't see anything for forcing a backdrop root: https://drafts.fxtf.org/filter-effects-2/#BackdropRootTriggers.

Reply:
Done (as best as we can for now).